sefaria/model/text.py

```diff
@@ -1686,6 +1739,76 @@ def __call__(cls, *args, **kwargs):
             return super(TextFamilyDelegator, cls).__call__(*args, **kwargs)
 
 
+class TextRange:
+
+    def __init__(self, oref, lang, vtitle, merge_versions=False):
+        if isinstance(oref.index_node, JaggedArrayNode) or isinstance(oref.index_node, DictionaryEntryNode): #text cannot be SchemaNode
+            self.oref = oref
+        elif oref.has_default_child(): #use default child:
+            self.oref = oref.default_child_ref()
+        else:
+            raise InputError("Can not get TextRange at this level, please provide a more precise reference")
+        self.lang = lang
+        self.vtitle = vtitle
+        self.merge_versions = merge_versions
+        self._versions = []
+        self._text = None
+        self.sources = None
+
+    @property
+    def versions(self):
+        if self._versions == []:
+            condition_query = self.oref.condition_query(self.lang) if self.merge_versions else \
+                {'title': self.oref.index.title, 'actualLanguage': self.lang, 'versionTitle': self.vtitle}
```

Comment:
I may be missing something but where is the lang param being correctly funneled to the correct database field (i.e. not original "language" that is only he/en)

Reply:
here - it uses 'actualLanguage' and not the he/en 'language'